test/metabase/test/data/sql/ddl.clj

```diff
@@ -52,9 +52,11 @@
             (sql.tx/create-table-sql driver dbdef tabledef)))
     ;; Add the SQL for adding FK constraints
     (doseq [{:keys [field-definitions], :as tabledef} table-definitions
-            {:keys [fk], :as fielddef}                field-definitions]
+            {:keys [fk indexed?], :as fielddef}                field-definitions]
       (when fk
-        (add! (sql.tx/add-fk-sql driver dbdef tabledef fielddef))))
+        (add! (sql.tx/add-fk-sql driver dbdef tabledef fielddef)))
+      (when indexed?
+        (add! (sql.tx/create-index-sql driver (:table-name tabledef) [(:field-name fielddef)]))))
```

Comment:
Adding this to the FK `doseq` block is inconsistent with the way column comments are added on lines 64-68 below, which has its own separate `doseq`. I don't care either way, but as long as its consistent it'll be easier to read and understand